copt/constraint.py

```diff
@@ -278,14 +278,16 @@ class TraceBall:
         alpha: float
             radius of the ball.
 
+        use_eigs: use an eigenvalue solver rather than an SVD solver. This
```

Comment:
Is this necessary? under the hood svds is using the same eigensolver you'r using below (well, almost, it's using the Hermitian version eigsh) (https://github.com/scipy/scipy/blob/v1.6.2/scipy/sparse/linalg/eigen/arpack/arpack.py#L1727-L1913) 

If the issue is that the tolerance is too low, I think we should use the tolerance parameter in svds instead

Reply:
Interesting. Maybe the tolerance was the issue! I found that they gave quite different solutions, with eigs reproducing what was in their paper and svds not. I was also thinking that it was strange though. Hopefully, it's the tolerance.